espressif/vscode-esp-idf-extension
# fix jtag flash on build flash monitor

## Description

Fix Build Flash Monitor command for JTAG flash use case.

Fixes #1235

## Type of change

Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "ESP-IDF: Build, Flash and start a monitor on your device" after setting JTAG flash type.
2. Execute action.
3. Observe results. You should see build, flash and monitor outputs in order as expected.

## How has this been tested?

Manual testing using "ESP-IDF: Build, Flash and start a monitor on your device" and "ESP-IDF: Flash (with JTag)"

**Test Configuration**:
* ESP-IDF Version: 5.2.2
* OS (Windows,Linux and macOS): macOS

## Checklist
